doc/developer_guide/Menu/CloseableMenu.adoc

```diff
@@ -35,4 +35,8 @@ public class MyNewMenu : CloseableMenu
 
 * Reference your menu GameObject under `Menu Object`
 
+* To close your new menu, reference the `Hide()` method in your close button
+
+* To show a submenu, reference the `Hide()` method of the new submenu in your button, to show the menu
```

Comment:
Alright, that's good, but you could elaborate on which menu to reference and maybe add a screen shot?

And secondly I think it should be `Show()`
```suggestion
* To close your new menu, reference the `Hide()` method in your close button

* To show a submenu, reference the `Show()` method of the new submenu in your button, to show the menu
```